MIPs/mip-5.md

```diff
@@ -135,7 +135,19 @@ Multichain API connections will be established and updated through [CAIP-25](htt
 
 > **Note:** MetaMask treats `requiredScopes` as `optionalScopes`. Only `optionalScopes` are recommended, though `requiredScopes` can be used to signal that your dapp will not be usable if certain [CAIP-217](https://chainagnostic.org/CAIPs/caip-217) `scopeStrings` are not authorized. 
 
-> **Note:** Developers are encouraged to precisely request only the authorization scopes for methods and notifications that their dapp expects to call before making additional `wallet_createSession` calls to expand authorization scopes. Requesting specific authorization scopes allows wallets to discover and implement features that are being adopted. Wallets can also further optimize permission confirmation flows to reduce unnecessary friction for some method calls. For simplicity, however, MetaMask may return more authorization scopes, methods, or notifications than the caller explicitly requested.
+> **Note:** Developers are encouraged to precisely request only the authorization scopes for methods and notifications that their dapp expects to call before making additional `wallet_createSession` calls to expand authorization scopes. Requesting specific authorization scopes allows wallets to discover and implement features that are being adopted. Wallets can also further optimize permission confirmation flows to reduce unnecessary friction for some method calls. For efficiency, however, MetaMask may return more authorization scopes, methods, or notifications than the caller explicitly requested.
+
+#### Supported `scopeObject` properties
+
+Supported scopeObject properties include:
+- `references`
+- `methods`
+- `notifications`
+- `accounts`
+
+> **Note:** The `references` parameter is mainly included as a shorthand when there would otherwise be repetitive `scopeObjects` with the only difference being the `reference` portion of each `scopeString`.
```

Comment:
thoughts on not mentioning references at all? It was meant as a shorthand convenience, but nobody should be manually forming CAIP-25 requests. Personally, I think it would be better simply to never advertise this so we don't get stuck having to support it forever. I'm unsure if we already expose this in the docs though. @adonesky1

Reply:
agreed. Lets not advertise `references!